Tests/FunctionalTests/Sources/FTRNetworkTest.m

```diff
@@ -29,6 +29,10 @@ - (void)setUp {
 }
 
 - (void)testSynchronizationWorksWithNSURLConnection {
+// This test uses APIs that don't exist when deployment target is 9.0 or greater.
+#if defined(__IPHONE_9_0) && (__IPHONE_OS_VERSION_MIN_REQUIRED >= __IPHONE_9_0)
```

Comment:
ditto

Reply:
I removed this ifdef, by suppressing the deprecation error, this test now works.